src/components/composer/content_editable_helper.ts

```diff
@@ -128,6 +129,21 @@ export class ContentEditableHelper {
       current = it.next();
     }
 
+    if (current.value === nodeToFind) {
+      if (!current.value.hasChildNodes()) {
+        usedCharacters += nodeOffset;
+      } else {
+        const children = [...current.value.childNodes].slice(0, nodeOffset);
+        usedCharacters += children.reduce((acc: number, child: Node) => {
+          if (child.textContent) {
+            return acc + child.textContent.length;
+          } else {
+            return acc;
+          }
+        }, 0);
```

Comment:
```suggestion
        for(const child of children){
            if (!child.textContent) continue;
            usedCharacters += child.textContent.length;
        };
```
Cleaner than a reduce, no ?

Reply:
Agreed, it sems to makes sense in 16.2 but it seems clearer